frontend/ui/widget/inputdialog.lua

```diff
@@ -352,18 +362,9 @@ function InputDialog:init()
         margin = self.input_margin,
         input_type = self.input_type,
         text_type = self.text_type,
-        enter_callback = self.enter_callback or function()
-            for _,btn_row in ipairs(self.buttons) do
-                for _,btn in ipairs(btn_row) do
-                    if btn.is_enter_default then
-                        btn.callback()
-                        return
-                    end
-                end
-            end
-        end,
+        enter_callback = not self.allow_newline and self.enter_callback,
         strike_callback = self.strike_callback,
-        edit_callback = self._buttons_edit_callback, -- nil if no Save/Close buttons
+        edit_callback = self._buttons_edit_callback or self.edited_callback,
```

Comment:
Not accurate anymore to keep that comment ?

Reply:
Fixed.